metaswitch/common/pdlogs.py

```diff
@@ -67,3 +69,21 @@ def log(self, **kwargs):
         Note that users should call syslog.openlog before calling this function,
         to set an appropriate process name."""
         syslog.syslog(self._priority, self._text.format(**kwargs))
+
+CONNECTION_LOST = PDLog(
+    number=PDLog.CL_PYTHON_COMMON_ID + 1,
+    desc="The connection to Cassandra has been lost.",
```

Comment:
I'm guessing this is currently fine because the only connections the python CommunicationMonitor monitors are connections to Cassandra, but it doesn't feel great going forward.

Can we either create Cassandra specific logs, and create the CommunicationMonitor with something identifying this log (as we do for alarms), or create the CommunicationMonitor with a string identifying the process it's monitoring, and substitute this string into the log when we raise it?